1-org/modules/centralized-logging/variables.tf

```diff
@@ -0,0 +1,235 @@
+  type        = bool
+  default     = false
+}
+
+variable "exclusions" {
+  description = "(Optional) A list of sink exclusion filters."
+  type = list(object({
+    name        = string,
+    description = string,
+    filter      = string,
+    disabled    = bool
+  }))
+  default = []
+}
+
+variable "bigquery_options" {
+  description = "(Optional) Options that affect sinks exporting data to BigQuery. use_partitioned_tables - (Required) Whether to use BigQuery's partition tables. Applies to logging target type: bigquery."
+  type = object({
+    use_partitioned_tables = bool
+  })
+  default = null
+}
+
+variable "labels" {
+  description = "(Optional) Labels attached to logging resources."
+  type        = map(string)
+  default     = {}
+}
+variable "kms_key_name" {
+  description = "(Optional) ID of a Cloud KMS CryptoKey that will be used to encrypt the logging destination. Applies to logging target types: bigquery, storage, and pubsub."
+  type        = string
+  default     = null
+}
+
+variable "logging_location" {
+  description = "(Optional) The location of the logging destination. Applies to logging target types: bigquery and storage."
+  type        = string
+  default     = "US"
+}
+
+variable "delete_contents_on_destroy" {
+  description = "(Optional) If set to true, delete all contained objects in the logging destination. Applies to logging target types: bigquery and storage."
+  type        = bool
+  default     = false
+}
+
+#----------------------------- #
+# Logbucket specific variables #
+#----------------------------- #
+variable "retention_days" {
+  description = "(Optional) The number of days data should be retained for the log bucket. Applies to logging target type: logbucket."
+  type        = number
+  default     = 30
+}
+
+#----------------------------- #
+# Big Query specific variables #
+#----------------------------- #
+variable "dataset_description" {
+  description = "(Optional) A user-friendly description of the dataset. Applies to logging target type: bigquery."
+  type        = string
+  default     = ""
+}
+
+variable "expiration_days" {
+  description = "(Optional) Table expiration time. If null logs will never be deleted. Applies to logging target type: bigquery."
+  type        = number
+  default     = null
+}
+
+#--------------------------- #
+# Storage specific variables #
+#--------------------------- #
+variable "storage_class" {
+  description = "(Optional) The storage class of the storage bucket. Applies to logging target type: storage."
+  type        = string
+  default     = "STANDARD"
+}
+
+variable "uniform_bucket_level_access" {
+  description = "(Optional) Enables Uniform bucket-level access to a bucket. Applies to logging target type: storage."
+  type        = bool
+  default     = true
+}
+
+variable "lifecycle_rules" {
```

Comment:
Let's not expose any vars we dont need in foundation currently to keep the interface smaller.